### PR TITLE
fix (TSEO-2): Issue where SEO description is not showing up on articl…

### PR DESCRIPTION
**Pull request by virendrapatelnuk**
…e pages

Issue where SEO description is not showing up on article pages

### CODE REVIEW

File: packages/article-skeleton/src/head.web.js [hunk 1]
```diff
@@ -102,7 +102,7 @@ function Head({ article, logoUrl, paidContentClassName }) {
 
   const publication = PUBLICATION_NAMES[publicationName];
   const authorName = getAuthorAsText(article);
-  const desc =
+  const desc = article.seoDescription ? article.seoDescription :
```
Comment (vinhlam) [inline packages/article-skeleton/src/head.web.js:105]: Feels like something that requires a unit test. This is also breaking linting, so make sure you fix that as well.

Comment (vinhlam) [reply packages/article-skeleton/src/head.web.js]: change for linting good, but I am expected a test in `head.test.web.js` with something that covers your new case.
eg.
```
    const testRenderer = TestRenderer.create(
      <Head
        article={{...article, seoDescription:'somevalue' }}
```

Comment (virendrapatelnuk) [reply packages/article-skeleton/src/head.web.js]: ok got it.

**Review by flashcheeks — APPROVED** [review]

File: packages/article-skeleton/src/head.web.js (hunk 1 above)
Comment (flashcheeks) [reply packages/article-skeleton/src/head.web.js]: Very minor: You could drop `article` from `article.seoDescription` if you add `seoDescription` to the destructuring expression above.

File: packages/article-skeleton/src/head.web.js [hunk 2]
```diff
@@ -102,10 +102,8 @@ function Head({ article, logoUrl, paidContentClassName }) {
 
   const publication = PUBLICATION_NAMES[publicationName];
   const authorName = getAuthorAsText(article);
-  const desc =
-    Array.isArray(descriptionMarkup) && descriptionMarkup.length
-      ? renderTreeAsText({ children: descriptionMarkup })
-      : null;
+  const desc = article.seoDescription || (Array.isArray(descriptionMarkup) && descriptionMarkup.length
```
Comment (flashcheeks) [inline packages/article-skeleton/src/head.web.js:105]: Very minor: You could drop article from article.seoDescription if you add seoDescription to the destructuring expression above.

**Review by vinhlam — APPROVED** [review]

File: packages/article-skeleton/src/head.web.js (hunk 2 above)
Comment (virendrapatelnuk) [reply packages/article-skeleton/src/head.web.js]: yes! thanks, @flashcheeks  I'll remember this next time

Comment (virendrapatelnuk) [reply packages/article-skeleton/src/head.web.js]: fixed

**Pull request merged**